modin/pandas/utils.py

```diff
@@ -113,6 +113,33 @@ def is_scalar(obj):
     return not isinstance(obj, BasePandasDataset) and pandas_is_scalar(obj)
 
 
+def is_full_grab_slice(slc, sequence_len=None):
+    """
+    Check that the passed slice grabs the whole sequence.
+
+    Parameters
+    ----------
+    slc : slice
+        Slice object to check.
+    sequence_len : int, optional
+        Length of the sequence to index with the passed `slc`.
+        If not specified the function won't consider `slc` to
+        be a full-grab if its ``.stop`` attribute is equal or
+        greater than the sequence length.
```

Comment:
wait, how the function can know sequence length if it's `None` to compare `slc.stop` to it?.. something's wrong with this docstring, pls fix

Reply:
Changed to the following:
"If not specified the function won't be able to check whether ``slc.stop`` is equal or greater than the sequence length to consider `slc` to be a full-grab, and so, only slices with ``.stop is None`` are considered to be a full-grab."